### PR TITLE
Feature proposal : Add docker usage

### PR DESCRIPTION
**Pull request by jaymoulin**
This feature add docker to ease installation and propose an isolated system. This is completely optional since I propose this on my own (https://github.com/jaymoulin/docker-coinmon) and images are already compiled.

### CODE REVIEW

**Comment by bichenkk** [comment]
@jaymoulin Thanks a lot! I was planning to do it this later! Let me check and I will merge it soon :)

**Comment by bichenkk** [comment]
hi @jaymoulin I have seen the file changes. Is that it should be coinmon instead of coinbase?

**Comment by jaymoulin** [comment]
You're right. That's why one should not work at night -_-

File: README.md [hunk 1]
```diff
@@ -85,6 +85,40 @@ $ npm link
 $ coinmon
 ```
 
+## Docker
+
+### Initial usage
+
+```
+$ docker run --rm -ti jaymoulin/coinmon
```
Comment (DavidLiedle) [inline README.md]: This is cool, but needs one additional step to address security via transparency: automating the Docker Hub image build via integration with GitHub. This allows users to inspect the Dockerfile used to generate the image they're pulling from your account, verifying that it contains exactly and only the lines contained in the Dockerfile distributed with the source of this project.

Comment (jaymoulin) [reply README.md]: I don't think this is the right place for this comment as @bichenkk don't maintain It. I suggest you open Issue here : https://github.com/jaymoulin/docker-coinmon/issues.

Nevertheless, I don't think i'll be able to automate build process due to the crossbuilding system, but you can consult dockerfile here : https://github.com/jaymoulin/docker-coinmon/raw/master/Dockerfile

**Pull request merged**